localstack-core/localstack/utils/docker_utils.py

```diff
@@ -153,10 +154,14 @@ def container_ports_can_be_bound(
             ports=port_mappings,
             remove=True,
         )
+    except DockerNotAvailable as e:
+        LOG.warning("Cannot perform port check because Docker is not available.")
```

Comment:
Not sure what's the best logging strategy if the exception is caught upstream as well 🤔